docs/core/testing/index.md

```diff
@@ -49,7 +49,7 @@ dependencies on infrastructure. These tend to make tests slower and
 more brittle, and thus should be reserved for integration tests. You
 can avoid these hidden dependencies in your application code by following
 the [Explicit Dependencies Principle](http://deviq.com/explicit-dependencies-principle/)
-and using [Dependency Injection](https://docs.asp.net/en/latest/fundamentals/dependency-injection.html)
+and using [Dependency Injection](https://docs.asp.net/en/latest/platform/dependency-injection.html)
```

Comment:
this new link doesn't work. maybe you should link between repos like this: /aspnet/core/fundamentals/dependency-injection